tensorflow/c/c_api.h

```diff
@@ -125,6 +125,14 @@ TF_CAPI_EXPORT extern void TF_DeleteBuffer(TF_Buffer*);
 
 TF_CAPI_EXPORT extern TF_Buffer TF_GetBuffer(TF_Buffer* buffer);
 
+// --------------------------------------------------------------------------
+// Used to pass strings across the C API. The caller does not take ownership 
```

Comment:
Should this say "return strings" specifically? Otherwise it is not clear which side is the caller in the next sentence. Does caller create and pass this object or does the caller get it as an output?